config/locales/en.yml

```
@@ -1,11 +1,23 @@
-# See https://github.com/svenfuchs/rails-i18n/tree/master/rails%2Flocale for starting points.
-
+# See https://github.com/svenfuchs/rails-i18n/tree/master/rails%2Flocale for starting
 en:
   dashboard:
-    _activity:
+   _activity:
```

```suggestion
   _activity:
```
How about introducing a space ' ' before line 3 such that relative 2 space indentation is maintained between lines. @Thekiddiejsandpython everything else is  good. Thanks!

---

Thank you so much! 😃